client/services/code_evaluators/CodeRunnerDispatcherServiceSpec.js

```diff
@@ -31,6 +31,37 @@ describe('CodeRunnerDispatcherService', function() {
       };
       expect(errorFunction).toThrowError(Error);
     });
+
+    it('should throw a timeout error if passed infinitely looping code',
+      function() {
+        var code =
+        `while True:
+          print 'a'
+        `;
+
+        CodeRunnerDispatcherService.runCodeAsync("python", code).then(
+          function(rawCodeEvalResult) {
+            expect(rawCodeEvalResult.getErrorString()).toBe(
+              'TimeLimitError: Program exceeded run time limit. on line 2')
```

Comment:
Why is there a period in the middle of the sentence? 

Edit: on thinking about it more, it seems like the "on line 2" part of this doesn't make sense. Perhaps we should remove that, in this case?